src/lib/ngsi/ContextElement.cpp

```diff
@@ -111,6 +111,32 @@ std::string ContextElement::render(ConnectionInfo* ciP, RequestType requestType,
 
 /* ****************************************************************************
 *
+* ContextElement::toJson - 
+*/
+std::string ContextElement::toJson(ConnectionInfo* ciP, NotificationFormat notifyFormat)
```

Comment:
Is ciP actually used in the body of the method? It doesn't seem so...

(Note that most of the toJson() method doesn't use ciP, differenctly to render() familiy)